sharedUtils/prepareGranuleAccessParams.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Takes granule params an returns the params to be used in a CMR request
```

Comment:
Takes granule params **and** returns the params to be used in a CMR request

Reply:
🤦